### PR TITLE
Fixed flag deprecations and updated Hugo version to latest

### PR DESCRIPTION
**Pull request by mbianchidev**
Updated Hugo version from 0.122.2 to 0.133.0 - latest as of 21/08/2024

Flag deprecations fixed:
- `.Site.IsServer` was deprecated in Hugo v0.120.0 and will be removed in Hugo 0.134.0, updated flag to `hugo.IsServer` instead.
- `.Site.GoogleAnalytics` was deprecated in Hugo v0.120.0 and will be removed in Hugo 0.134.0, updated flag to `.Site.Config.Services.GoogleAnalytics.ID` instead.
- `.Site.DisqusShortname` was deprecated in Hugo v0.120.0 and will be removed in Hugo 0.134.0, updated flag to `.Site.Config.Services.Disqus.Shortname` instead.

Fixed the 'includes' bundle not found error for Hindi,
Fixed the the 'includes' bundle not found error for Russian.

### CODE REVIEW

**Comment by mbianchidev** [comment]
PTAL @kubernetes/website-maintainers - see [this discussion](https://kubernetes.slack.com/archives/C1J0BPD2M/p1724226266705639) on Slack for more information about this change.

**Review by dipesh-rawat — COMMENTED** [review]
The website appears to be working fine in the [build preview (here)](https://deploy-preview-47612--kubernetes-io-main-staging.netlify.app/docs/home/).
Thanks for these updates!

/lgtm

File: content/hi/includes/index.md
Comment (asem-hamid) [inline content/hi/includes/index.md:1]: Is this changes for all localize versions ??

File: netlify.toml
Comment (shurup) [inline netlify.toml:10]: We need to update this to resolve a conflict that arose after #47618 was merged.
```suggestion
NODE_VERSION = "20.17.0"
```

File: content/hi/includes/index.md
Comment (asem-hamid) [inline content/hi/includes/index.md:3]: Is this changes for all localize versions ??

**Comment by shurup** [comment]
Thank you for the great work done here! Let's resolve the conflict and merge it.

P.S. Also, on behalf of the Russian localisation team, I'm confirming that we're now aware of adding the `ru/includes/index.md` file.

**Review by Okabe-Junya — COMMENTED** [review]
Thank you for this great work!

On behalf of the Japanese l10n team, let me approve (Just Comment)
We have `content/ja/includes/index.md`, so you don't need any more work related to ja l10n

**Comment by mbianchidev** [comment]
Solved, it was actually my other PR getting merged #47618 @shurup 
Thanks for taking a look 👍 

Arigato @Okabe-Junya :)

**Review by Okabe-Junya — COMMENTED** [review]
/lgtm

**Comment by sftim** [comment]
If you want to squash this to 2 commits @mbianchidev, that'd be a nice tidying.

**Comment by mbianchidev** [comment]
> /label tide/merge-method-squash

Isn't tide squashing in this way? @sftim 
I genuinely don't know how the merge-method work

**Comment by shurup** [comment]
> Isn't tide squashing in this way?

Right, I made it for that specific reason.

**Comment by sftim** [comment]
If you can follow the Kubernetes guidance on [squashing](https://www.kubernetes.dev/docs/guide/pull-requests/#squashing), great @mbianchidev.

If not, that's OK - we can get GitHub to do a similar but less tidy squash (which is what that label signals Tide to do).

**Comment by mbianchidev** [comment]
> If you can follow the Kubernetes guidance on [squashing](https://www.kubernetes.dev/docs/guide/pull-requests/#squashing), great @mbianchidev.
> 
> If not, that's OK - we can get GitHub to do a similar but less tidy squash (which is what that label signals Tide to do).

Quoting the docs:
"Using this label can help when squashing by hand is considered too challenging or not worth the extra effort. It can also speed up merging because squashing by hand implies getting another LGTM from a reviewer and re-run of the CI tests."
In particular I find manually squashing not worth the extra effort for this one.

I could indeed squash myself via a simple
`git reset --soft HEAD~5 && git commit 'new message'` && git push --force` on my branch, don't get me wrong (I'm quite accustomed to git) but that would mean having 3 people LGTM (1 per localization + 1 extra) to then wait again for an approve label.
Since this PR is blocking another one with a shiny new feature for localized links that would be a great addition, I would rather have it merged in the fast and less-elegant way with an unpolished squash commit message.

For any next PR I'll do my best to avoid unmeaningful commits messages such as "Update netlify.toml" or "merge main into main", my bad for these ones.
Hope that is ok, let me know otherwise!

**Comment by mbianchidev** [comment]
> To be clear, this isn't waiting on you to do a squash and forced-push @mbianchidev; you can do that if you want to, but what it's actually waiting on is approval.
> 
> Prow preserves LGTM if you run git reset --soft HEAD~5 && git commit 'new message' && git push --force so you wouldn't need 3 new LGTMs.
> 
> Don't feel you have to do that squash here.

Noted on the prow functioning around resetting and force pushing in a fork, thanks for clarifying that.

I was still wrong about that command to run since the situation on [my fork's main branch](https://github.com/mbianchidev/kubernetes-website) is  slightly different than I expected - reverse timeline with the first items being the latest:
- 2 commits I did to solve the conflict
- A bunch of commits taken in from the rebase
- 3 commits that actually solve the issue

If you have any advice on how to untangle this and get a cleaner history, please let me know but I honestly wouldn't spend too much time on it for the reasons I explained above.
In the meantime I'll try to get the approval so we can unblock @shurup's PR!

**Comment by sftim** [comment]
It's trickier if you use the main branch in your fork.
For next time, if you're not working on the _main_ branch, try this oneliner:
```
git reset --soft "$(git merge-base main HEAD)" )" && git commit 'new message' && git push --force-with-lease
```

I don't know enough about your local clone to recommend the commands if you're working on _main_ in your fork.

&hellip;but it's still OK to leave the squash to GitHub.
(I'll still suggest the tidying in future PRs, from anyone, when I think it's beneficial)

---

This PR is ready for approval.

**Comment by mbianchidev** [comment]
> It's trickier if you use the main branch in your fork. For next time, if you're not working on the _main_ branch, try this oneliner:
> 
> ```
> git reset --soft "$(git merge-base main HEAD)" )" && git commit 'new message' && git push --force-with-lease
> ```
> 
> I don't know enough about your local clone to recommend the commands if you're working on _main_ in your fork.
> 
> …but it's still OK to leave the squash to GitHub. (I'll still suggest the tidying in future PRs, from anyone, when I think it's beneficial)
> 
> This PR is ready for approval.

I'll give the command a try next time, for sure, thanks a ton!

About the current main situation I guess I would've to rewrite the history completely, cherry-pick each of my commits to then squash them in 2 separated ones: "Updated Hugo version" and "Fixed Hindi and Russian bundle".
It would make sense but there's a certain degree of "risk" involved as in "oh no I didn't rewrite the history correctly, what now?"  😅 

You are totally right and I'm usually a fan of tidy commits and history but lazyness one this time around.
I'll seek the label now, thanks for giving this an informal one, appreciate it!

**Comment by Okabe-Junya** [comment]
Who would be the best to approve this PR?

Should we wait for the SIG-Docs TL or Chair? I know Tim has stepped down from being TL, but you are still a website-owner, which means if you approve it, this PR will be merged.

**Comment by sftim** [comment]
> Should we wait for the SIG-Docs TL or Chair? I know Tim has stepped down from being TL, but you are still a website-owner, which means if you approve it, this PR will be merged.

I decline to use the admin powers I still have; they are pending cleanup. Right now the only approvers for a Hugo version bump are the SIG leadership.

**Review by truongnh1992 — APPROVED** [review]
LGTM

**Comment by divya-mohan0209** [comment]
Thank you for these changes!
/approve

**Pull request merged**
